src/screens/UserManagement/UserRevamp/UserManagementUtils.res

```diff
@@ -130,11 +130,19 @@ let getUserManagementViewValues = (~checkUserEntity) => {
   }
 }
 
-let stringToVariantMapper = roleId => {
+let stringToVariantMapperInternalUser = roleId => {
   open UserManagementTypes
   switch roleId {
   | "internal_view_only" => InternalViewOnly
   | "internal_admin" => InternalAdmin
   | _ => NonInternal
   }
 }
+
+let stringToVariantMapperTenantAdmin = roleId => {
```

Comment:
Rather than opening for a single use case we can use in some what this way - Try this just a suggestion.

```
let stringToVariantMapperTenantAdmin: string => UserManagementTypes.admin = roleId =>
  switch roleId {
  | "tenant_admin" => TenantAdmin
  | _ => NonTenantAdmin
  }

```

Can be done above too as just return needs a type. Check once.